config/application.rb

```diff
@@ -32,5 +32,9 @@ class Application < Rails::Application
     config.api_only = true
 
     config.time_zone = 'UTC'
+
+    config.i18n.available_locales = [:de, :en, :fr, :it, :rm]
```

Comment:
If I understand the code correctly, the option for swiss-german is missing. Could it be added?

Reply:
How? What's the appropriate HTTP_ACCEPT_LANGUAGE header for schweizerdeutsch?

Reply:
[This site](https://www.niefuend.org/blog/internet/2017/10/alle-accept-language-codes-mit-laendernamen/) says it would be "de-ch". Using the[ ISO 639-1 up-to ISO 639-3](https://de.wikipedia.org/wiki/Schweizerdeutsch) would be "gsw". Another possibility could be just "ch" to match the style of the others, but I'm really not familiar with how this is usually handled 😬

Reply:
`de-CH` is german in Switzerland. That's for sure not what you intend. In BCP47's RFCs I didn't find any mapping to ISO, so I think `gsw` is the correct one according to the IANA tag list: http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry